.cache/01/b39e3fe37ba6634381d19259b8d4be.json

```diff
@@ -0,0 +1 @@
ource":"../node_modules/lodash/_createAggregator.js","original":{"line":16,"column":0},"generated":{"line":1,"column":169}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":17,"column":0},"generated":{"line":1,"column":171}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":17,"column":0},"generated":{"line":1,"column":173}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":17,"column":0},"generated":{"line":1,"column":175}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":17,"column":0},"generated":{"line":1,"column":179}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":19,"column":0},"generated":{"line":1,"column":182}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":19,"column":0},"generated":{"line":1,"column":189}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":19,"column":0},"generated":{"line":1,"column":191}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":19,"column":0},"generated":{"line":1,"column":193}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":19,"column":0},"generated":{"line":1,"column":195}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":19,"column":0},"generated":{"line":1,"column":197}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":19,"column":0},"generated":{"line":1,"column":199}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":19,"column":0},"generated":{"line":1,"column":202}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":23,"column":0},"generated":{"line":1,"column":206}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":23,"column":0},"generated":{"line":1,"column":213}},{"source":"../node_modules/lodash/_createAggregator.js","original":{"line":23,"column":0},"generated":{"line":1,"column":221}}],"sources":{"../node_modules/lodash/_createAggregator.js":"var arrayAggregator = require('./_arrayAggregator'),\n    baseAggregator = require('./_baseAggregator'),\n    baseIteratee = require('./_baseIteratee'),\n    isArray = require('./isArray');\n\n/**\n * Creates a function like `_.groupBy`.\n *\n * @private\n * @param {Function} setter The function to set accumulator values.\n * @param {Function} [initializer] The accumulator object initializer.\n * @returns {Function} Returns the new aggregator function.\n */\nfunction createAggregator(setter, initializer) {\n  return function(collection, iteratee) {\n    var func = isArray(collection) ? arrayAggregator : baseAggregator,\n        accumulator = initializer ? initializer() : {};\n\n    return func(collection, setter, baseIteratee(iteratee, 2), accumulator);\n  };\n}\n\nmodule.exports = createAggregator;\n"},"lineCount":null}},"error":null,"hash":"6a49c6be9d51e36c7fc32f21bc2937dd","cacheData":{"env":{}}}
```

Comment:
I think that `.cache` shouldn't be included in the git repository? Or is this needed for netlify?

You can make git ignore it like the node_modules by putting this is in `.gitignore` like so:
```diff
   # dependencies
   /node_modules

   # misc
   .DS_Store
   .env
+ .cache/
```